docs/en/development.rst

```diff
@@ -188,8 +188,8 @@ templates that bake uses. The best way to do this is:
 
 #. Bake a new plugin. The name of the plugin is the bake 'theme' name
 #. Create a new directory **plugins/[name]/src/Template/Bake/Template/**.
-#. Copy any templates you want to override from
-   **vendor/cakephp/bake/src/Template/Bake/Template** to matching files in your
+#. Copy any templates you want to override from 
+   **vendor/cakephp/bake/templates/bake/Template** to matching files in your
```

Comment:
```suggestion
   **vendor/cakephp/bake/templates/bake/** to matching files in your
```

There are more templates than just view templates.

Reply:
@markstory  

> #. Bake a new plugin. The name of the plugin is the bake 'theme' name
> #. Create a new directory **plugins/[name]/src/Template/Bake/Template/**.
> 

I haven't worked with plugins yet, so I can't say anything about it